### PR TITLE
Fixes #5388 resolves addon custom icon squashing

### LINKED ISSUE #5388: Add-on custom icons are not correctly displayed
Moving issue https://github.com/mozilla/addons-server/issues/7956

STR:
1. Go to Edit images in dev hub for a submitted ad-on
2. Upload a .jpg sample from edit images using upload a custom icon button
3. Save changes and click on View Listing
4. Observe the icon on the ad-on detail page

Actual result:
Images look a bit squashed.

Expected result:
Images are correctly displayed.

Notes:
This issue was reproducible for the user info card but then fixed [here](https://github.com/mozilla/addons-frontend/issues/4598).
This issue is reproducible with .jpg and .png samples with normal size and dimension, it is more obvious for images with a larger width.
This is reproducible on AMO dev with FF59(Win10).

picture squashed

### PR DESCRIPTION
**Pull request by ShivangiKakkar**
Fixes #5388 

Image used - size: 1.96 MB (JPEG), dimensions: 1980 × 1289

### BEFORE
screen shot 2018-08-17 at 2 27 43 am

### AFTER
screen shot 2018-08-17 at 3 09 46 am

### Description
- Wrapped the custom icon image inside a div `Addon-img-wrapper` and imposed the restriction of `height` and `width` acc to screen size. And, changed styles for class `Addon-icon-image` to `height:auto;` and `width:100%` thus resolving the squashing problem. 
- By the following changes, horizontal images do not squash anymore but if the height of image is greater than 64px(potrait images) its cropped from bottom (not squashed), which should be the desired behaviour as read in the #4629. Please confirm this? 
- Do I need to write tests for the same?

Thank you for all the help, please review this @willdurand  @ioanarusiczki

### CODE REVIEW

File: src/amo/components/Addon/index.js [hunk 1]
```diff
@@ -183,11 +183,13 @@ export class AddonBase extends React.Component {
 
     return (
       <div className="Addon-icon" key="Addon-icon-header">
-        <img
-          alt={label}
-          className="Addon-icon-image"
-          src={getAddonIconUrl(addon)}
-        />
+        <div className="Addon-img-wrapper">
```
Comment (willdurand) [inline src/amo/components/Addon/index.js]: Could we use the parent `div` (`.Addon-icon`) or do we need another `div` here? If we have to keep this new `div`, can we name it `Addon-icon-wrapper`?

Comment (ShivangiKakkar) [reply src/amo/components/Addon/index.js]: I guess we will not be able to apply the height and width constraints in the parent div.

Yes sure, I am changing the name to `Addon-icon-wrapper` and pushing it in a minute.
Thanks for the review!  :)

Comment (ShivangiKakkar) [reply src/amo/components/Addon/index.js]: P.S - I believe having an image cropper in place would have been the best solution! according to me, but that's totally another issue. :)

**Comment by ShivangiKakkar** [comment]
@willdurand, Thanks for the review!
What should I do in order to get this merged?

**Comment by willdurand** [comment]
Merged! Thanks @ShivangiKakkar

**Comment by caitmuenster** [comment]
Yay, thanks for the patch, @ShivangiKakkar! Your contribution has been added to our [recognition wiki](https://wiki.mozilla.org/Add-ons/Contribute/Recognition) and your mozillians.org profile has been vouched. Welcome aboard! 

Looking forward to seeing you around the project. :)

**Comment by ShivangiKakkar** [comment]
@caitmuenster,
 #### Thank you so much. :)

**Pull request merged**